resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -6,10 +6,11 @@
     executor:
       docker:
         image: "resolwebio/archive-samples"
-  version: 0.0.2
+  version: 0.0.3
   type: "data:archive:samples"
   category: other
   persistence: TEMP
+  scheduling_class: interactive
```

Comment:
> Users are waiting with an open dialog for the achieve

Exactly my point. So is it ok if they wait 30 seconds? 60 seconds? How long is acceptable for an interactive process? Currently, 30 seconds is the limit and anything longer than that is killed. We need to set a hard limit on CPU time for interactive processes, otherwise this will just be abused to get priority. cc @tjanez @dblenkus